src/app/modules/design-patterns/abstract-factory/interfaces/striker-player.interface.ts

```diff
@@ -0,0 +1,5 @@
+import { AbstractBasePlayer } from './base-player.interface';
+
+export interface IStrikerPlayer extends AbstractBasePlayer {
```

Comment:
the interface IStrikerPlayer should extends IBasePlayer(the base player contract)

Reply:
I don't have IBasePlayer itself... I have `AbstractBasePlayer` which has `name`, `photo`, and `tshirt`.
Do you mean the name of the class or a different class with other properties?